pkg/statistics/handle/cache/internal/lfu/lfu_cache.go

```
@@ -43,6 +44,11 @@ type LFU struct {
 	cost         atomic.Int64
 	closed       atomic.Bool
 	closeOnce    sync.Once
+	// waiting counts the WaitForAsyncUpdates calls that are inside Ristretto. Ristretto's Close
```

golangci-lint (2.13.2)

[warning] line-length-limit: line is 96 characters, out of limit 80

(revive)